src/trex/molecule.py

```diff
@@ -1,19 +1,49 @@
 """
 A molecule is a DNA/RNA fragment that has potentially been sequenced multiple times
 """
+import math
 from typing import NamedTuple, List
-from collections import defaultdict
+from collections import defaultdict, Counter
 import numpy as np
 
 from .bam import Read
 
+# Threshold for low-complexity cloneIDs
+MIN_SHANNON_ENTROPY = 1.0
```

Comment:
Let's open a different issue to further discuss if this should be fixed, selected by the user or calculated.